### DISCUSSION TITLE
Can submit a request

### DISCUSSION THREAD

**Discussion by itsh3sam** [community/community#57570] [GitHub Education]
### Hi everyone,

Today I'm trying really hard to get pass through "Upload proof of your academic status" page and I can't.
After the pre-check process, the page shockingly just refreshes and nothing else happens.
It's so annoying that it doesn't even throw an error, so I understand what's wrong with the photo it just stupidly keeps refreshing.
What can I do?

**Comment by samus-aran** [comment]
Hi @AnonC0DER  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
